packages/neos-ui/src/Containers/Modals/DeleteNode/index.js

```diff
@@ -47,20 +47,30 @@ export default class DeleteNodeModal extends PureComponent {
     }
 
     renderTitle() {
-        const {nodeToBeDeletedContextPath, getNodeByContextPath, nodeTypesRegistry} = this.props;
-        const node = getNodeByContextPath(nodeToBeDeletedContextPath);
-        const nodeType = $get('nodeType', node);
-        const nodeTypeLabel = $get('ui.label', nodeTypesRegistry.get(nodeType)) || 'Neos.Neos:Main:node';
-
+        const {nodesToBeDeletedContextPaths, getNodeByContextPath, nodeTypesRegistry} = this.props;
+        if (nodesToBeDeletedContextPaths.length === 1) {
+            const singleNodeToBeDeletedContextPath = nodesToBeDeletedContextPaths[0];
+            const node = getNodeByContextPath(singleNodeToBeDeletedContextPath);
+            const nodeType = $get('nodeType', node);
+            const nodeTypeLabel = $get('ui.label', nodeTypesRegistry.get(nodeType)) || 'Neos.Neos:Main:node';
+            return (
+                <div>
+                    <Icon icon="exclamation-triangle"/>
+                    <span className={style.modalTitle}>
+                        <I18n id="Neos.Neos:Main:delete" fallback="Delete"/>
+                        &nbsp;
+                        <I18n id={nodeTypeLabel} fallback="Node"/>
+                        &nbsp;
+                        "{$get('label', node)}"
+                    </span>
+                </div>
+            );
+        }
         return (
             <div>
                 <Icon icon="exclamation-triangle"/>
                 <span className={style.modalTitle}>
-                    <I18n id="Neos.Neos:Main:delete" fallback="Delete"/>
-                    &nbsp;
-                    <I18n id={nodeTypeLabel} fallback="Node"/>
-                    &nbsp;
-                    "{$get('label', node)}"
+                    Delete {nodesToBeDeletedContextPaths.length} nodes
```

Comment:
We should use labels here.

Reply:
Ouuuch, good catch, I thought I had replaced all of those, will fix